app/utils/versionChecker.js

```diff
@@ -1,18 +1,15 @@
 class VersionChecker {
-  latest (callback) {
```

Comment:
Could you please explain rationale behind this change? so I understand it better.